src/supporting-data/database/components/entry/Entry.tsx

```diff
@@ -0,0 +1,68 @@
+import { RouteChildrenProps } from 'react-router-dom';
+import { Loader, Card } from 'franklin-sites';
+
+import SingleColumnLayout from '../../../../shared/components/layouts/SingleColumnLayout';
+import ErrorHandler from '../../../../shared/components/error-pages/ErrorHandler';
+import RenderColumnsInCard from '../../../../shared/components/results/RenderColumnsInCard';
+
+import useDataApi from '../../../../shared/hooks/useDataApi';
+
+import apiUrls from '../../../../shared/config/apiUrls';
+
+import { Namespace } from '../../../../shared/types/namespaces';
+import { DatabaseAPIModel } from '../../adapters/databaseConverter';
+import DatabaseColumnConfiguration, {
+  DatabaseColumn,
+} from '../../config/DatabaseColumnConfiguration';
+
+const DatabaseEntry = (props: RouteChildrenProps<{ accession: string }>) => {
+  const accession = props.match?.params.accession;
+
+  const {
+    data,
+    loading,
+    error,
+    status,
+    progress,
+  } = useDataApi<DatabaseAPIModel>(
+    apiUrls.entry(accession, Namespace.database)
+  );
+
+  if (error || !accession) {
+    return <ErrorHandler status={status} />;
+  }
+
+  if (loading || !data) {
+    return <Loader progress={progress} />;
+  }
+
+  return (
+    <SingleColumnLayout>
+      <h2>Database - {data.abbrev}</h2>
+      <Card>
+        <RenderColumnsInCard
+          renderers={DatabaseColumnConfiguration.get(DatabaseColumn.name)}
```

Comment:
A potentially premature comment: could simplify the code a bit here by mapping over `[DatabaseColumn.name, DatabaseColumn.server,...]` and calling `RenderColumnsInCard`.

Reply:
Reused same code everywhere, but I'm expecting this to be changed/overriden when handling the "card content" subtask in each of the 6 supporting data jiras. Not sure RenderColumnsInCard is the best approach for these, as the mockup requirements are slightly different (for example, full blue bar on the left, or 2 columns within the card for some namespaces like taxonomy).